<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pageMaterial</class>
 <widget class="QWidget" name="pageMaterial">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>677</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>659</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 18px;
font-weight: bold;
color: #1E3A5F;
background-color: #F4F5F7;
border-radius: 8px;</string>
     </property>
     <property name="text">
      <string>Materials</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="tableMaterial">
     <property name="styleSheet">
      <string notr="true">QTableWidget {
	background-color: #ffffff;
	alternate-background-color: #f2f5fa;
	gridline-color: #d0d4da;
	font-size: 13px;
	border: 1px solid #ccc;
}

QHeaderView::section {
	background-color: #e6e9ef;
	font-weight: bold;
	font-size: 14px;
	color: #333333;
	padding: 4px;
	border: 1px solid #d0d4da;
}</string>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
     </property>
     <property name="columnCount">
      <number>4</number>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Material id</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Unit</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Quantity in stock</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="horizontalLayout" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: #3A5F8F;
	color: #FFFFFF;
	font-size: 13px;
	font-weight: bold;
	border-radius: 4px;
	padding: 6px 10px;
}

QPushButton:hover {
	background-color: #2E4B70;
}

QPushButton:pressed {
	background-color: #1F344D;
}

QPushButton:pressed {
	background-color: #2a5d8f;
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="btn_addMaterial">
        <property name="text">
         <string>Add</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_editMaterial">
        <property name="text">
         <string>Edit</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_deleteMaterial">
        <property name="text">
         <string>Delete</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
